cli/command/formatter/sorter.go

```diff
@@ -0,0 +1,321 @@
+package formatter
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+	"time"
+)
+
+const (
```

Comment:
I think this is a little bit different, this is a keyword, so it has to be string.

It is supposed to appear in `--sort-by createdat:asc` or `--sort-by repository:desc`, so here we can't convert it to bool.